front50-proto/front50-proto.gradle

```diff
@@ -0,0 +1,59 @@
+buildscript {
+  repositories {
+    jcenter()
+    maven { url "https://spinnaker.bintray.com/gradle" }
+    maven { url "https://plugins.gradle.org/m2/" }
+  }
+  dependencies {
+    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.8'
+  }
+}
+
+apply plugin: 'java'
+apply plugin: 'com.google.protobuf'
+
+dependencies {
+  // TODO(rz): Promote these dep versions into kork-dependencies
+  implementation 'com.google.protobuf:protobuf-java:3.0.0'
+  implementation 'com.google.guava:guava:23.5-jre'
+  api 'com.google.api.grpc:grpc-google-common-protos:1.0.5'
+  implementation 'io.grpc:grpc-all:1.8.0'
+}
+
+protobuf {
+  protoc {
+    artifact = 'com.google.protobuf:protoc:3.5.1-1'
+  }
+  plugins {
+    grpc {
+      artifact = "io.grpc:protoc-gen-grpc-java:1.9.0"
+    }
+  }
+
+  generateProtoTasks {
+    ofSourceSet('main').each { task ->
+      task.builtins {
+        java{
+          outputSubDir = 'java'
+        }
+      }
+      task.plugins {
+        grpc {
+          outputSubDir = 'java'
+        }
+      }
+      task.descriptorSetOptions.includeImports = true
+    }
+  }
+  generatedFilesBaseDir = "${projectDir}/build-gen/proto"
+}
+
+idea {
+  module {
+    sourceDirs += file("${projectDir}/build-gen/proto/main/java")
+  }
+}
+
+clean {
```

Comment:
Using the `generatedSourcesDirs` above means that the build in `clean` task would work as usual - no need to for this